conf/modules/copilot.xml

```diff
@@ -35,7 +35,27 @@
   <datalink message="COPILOT_STATUS_DL" fun="copilot_parse_copilot_status_dl(buf)"/>
   <datalink message="MOVE_WP" fun="copilot_parse_move_wp_dl(buf)"/>
   <makefile target="ap">
-    <file name="copilot.c"/>
+    <file name="copilot_common.c"/>
+      <raw>
+        # attach the correct file based on firmware
+        RESULT=0
+
+        ifneq (,$(findstring ROTORCRAFT,$($(TARGET).CFLAGS)))
```

Comment:
I noticed it too late, fixed it with https://github.com/paparazzi/paparazzi/commit/e30a03cc0073b55a5a2ff59fe4ec298c810d22d0